### ISSUE TITLE
Scope

### ISSUE THREAD

**Issue by dann95** [GonzagaAccess/facebook#1]
let's use this issue to define the scope?

ping @RafaelGSS

**Comment by RafaelGSS** [comment]
Sure!

The project is about:

Create a CLI Cross-platform to manage your account on facebook that makes sense for accessibility, with these features:

- Read feed
- Publish message

I already create an issue inside **NVDA** to better integration with screen readers.
[here](https://github.com/nvaccess/nvda/issues/10197)

**Comment by dann95** [comment]
right, what about the intended platforms? and tech to build the application?

**Comment by RafaelGSS** [comment]
I think that any language of script/cross-platform is okay for this project... NodeJS, Ruby, Python

Just need to make the more accessible toolkit software, for works with NVDA, Virtual Vision and another's screen readers... need many e2e tests to it and a good pipeline.

Obs: Is a good start to make a roadmap of the project, right? Maybe a illustration

**Comment by dann95** [comment]
i've never built something CLI tought node and pack it to executable, almost users uses windows, so guess it's the best platform to use as "main compatibility", also, in what platforms that NVDA works?

**Comment by dann95** [comment]
Probably facebook apis that allow to post messages, need to authenticate tought oauth2, right? how it gonna be "accessible" ?

**Comment by RafaelGSS** [comment]
> i've never built something CLI tought node and pack it to executable, almost users uses windows, so guess it's the best platform to use as "main compatibility", also, in what platforms that NVDA works?

This can help us https://github.com/zeit/pkg.
NVDA runs on Windows and most of the users are on windows.

> Probably facebook apis that allow to post messages, need to authenticate tought oauth2, right? how it gonna be "accessible" ?

Yep! The OAuth2 of facebook is accessible. I think that the session cache is large... However, I don't see large problems with OAuth2, Is like to add third parts app to Facebook, right?

**Comment by dann95** [comment]
I see no problem in oauth2, i just ask if it can be running with "screen reader".

**Comment by RafaelGSS** [comment]
Yes can! We application do not display sounds like a screen reader, so both should work together.

**Comment by RafaelGSS** [comment]
So, for now, what is a best getting started?

- Roadmap
- Example visual
- Others...

**Comment by dann95** [comment]
> So, for now, what is a best getting started?
> 
> * Roadmap
> * Example visual
> * Others...

* feature list

**Comment by RafaelGSS** [comment]
I think about:

- Login with Facebook OAuth2
- Publish text to timeline
- Read user timeline

For first release, what do you think?

**Comment by dann95** [comment]
enumarate groups? friends? see user/group timeline?

**Comment by RafaelGSS** [comment]
Maybe to next release... I think that principal features are above

**Comment by RafaelGSS** [comment]
Can you initialize the project with PR? I think that NodeJS sounds good

**Comment by dann95** [comment]
i can, but only on weekend.

**Comment by RafaelGSS** [comment]
Thanks! Maybe I started it before

Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by RafaelGSS** [comment]
I making some tests with NVDA and CLI Nodejs... However, I think that we need more projects because maybe theses tests can consume a long time,  do you have an idea? @dann95
